### PR TITLE
Use UUID for job and command identifiers

### PR DESCRIPTION
**Pull request by SkyToGround**
As the title says.

### CODE REVIEW

File: file_writer_control/InThreadStatusTracker.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     FILE_IDENTIFIER as STOPPED_IDENTIFIER,
 )
 from datetime import datetime, timedelta
-from streaming_data_types.utils import _get_schema
+from streaming_data_types.utils import get_schema
```
Comment (SkyToGround) [inline file_writer_control/InThreadStatusTracker.py:9]:
```suggestion
from streaming_data_types.utils import _get_schema
```

File: file_writer_control/InThreadStatusTracker.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def process_message(self, message: bytes):
         Process a binary message.
         :param message: The binary message to be processed.
         """
-        current_schema = _get_schema(message).encode("utf-8")
+        current_schema = get_schema(message).encode("utf-8")
```
Comment (SkyToGround) [inline file_writer_control/InThreadStatusTracker.py:56]:
```suggestion
        current_schema = _get_schema(message).encode("utf-8")
```

**Review by umithardal — APPROVED** [review]

**Comment by umithardal** [comment]
Just out of curiosity, do we need a human-readable form of command-id?

**Comment by SkyToGround** [comment]
@umithardal Short answer: no.

**Pull request merged**